docs/installation.md

```diff
@@ -11,14 +11,40 @@ In order to install an epoch node using a release binary, you need to:
 
 The release binaries are published on [GitHub](https://github.com/aeternity/epoch/releases) and are tested on the following platforms:
 * Ubuntu 16.04.3 LTS (x86-64);
+* Ubuntu 18.04 LTS (x86-64);
 * macOS Sierra (x86-64);
 * macOS High Sierra (x86-64).
 
 ## Install dependencies
 
-### Ubuntu 16.04 package
+### Ubuntu package
 
-The Ubuntu 16.04 package requires a libsodium shared library (v1.0.16) in `/usr/local/lib/libsodium.so.23` (`wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.16.tar.gz`, unpack, then do `./configure && make && sudo make install && sudo ldconfig`).
+The package requires a libsodium v1.0.16 as `libsodium.so.23` shared object/library.
+
+#### Ubuntu 18.04
+
+Ubuntu 18.04 ships with libsodium 1.0.16, thus it can be installed with `apt` package manager:
+
+```bash
+sudo apt-get install libsodium23
```

Comment:
I am considering how to reduce the testing effort for this user experience i.e. package built on 16.04 but run on 18.04. May I suggest to update the Dockerfile in this epoch repo so that, when releasing, the testing of the Docker image implicitly tests this? (The testing that the package built on 16.04 can be run on 16.04 is on testnet nodes.)

I see this PR does not change the Dockerfile in this epoch repo, that builds package on aeternity/builder - that is based on ubuntu:16.04 - and then actually prepares the image from ubuntu:16.04 .
Can you confirm that [the copying of `/usr/local/lib` from the builder](https://github.com/aeternity/epoch/blob/3019a2bf09e4da87fa8ae45e974984904ec02d2b/Dockerfile#L18) is only for [libsodium](https://github.com/aeternity/docker-builder/blob/a143797b693c31ba928b4c60390859e33b151225/Dockerfile#L22-L28)?

Reply:
Please see 3821683a